assets/src/components/v2/dup/dup_rotation_wrapper.tsx

```diff
@@ -0,0 +1,16 @@
+import React, { ReactNode } from "react";
+
+export const splitRotationFromPropNames = (WrappedComponent: React.ElementType, rotation: string) => {
+  return class extends React.Component {
+    render() {
+
+      const modifiedProps = Object.keys(this.props).reduce((newProps: any, currentKey: string) => {
+        const newKey = currentKey.replace("_one", "").replace("_two", "").replace("_zero", "")
```

Comment:
This should work, but since we only ever expect one of them at a time, what about ```replace(`_${rotation}`, "")``` as a more explicit option?